whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisRestServiceSSOAclTestIntegration.java

```diff
@@ -36,8 +32,10 @@ public class WhoisRestServiceSSOAclTestIntegration extends AbstractIntegrationTe
     private static final String LOCALHOST = "127.0.0.1";
     private static final String LOCALHOST_WITH_PREFIX = "127.0.0.1/32";
     public static final String VALID_TOKEN_USER_NAME = "person@net.net";
+    public static final String DB_E2E_1_USER_NAME = "db_e2e_1@ripe.net";
     public static final UserSession VALID_TOKEN_USER_SESSION = new UserSession("aff2b59f-7bd0-413b-a16f-5bc1c5c3c3ef","person@net.net", "Test User", true, "2033-01-30T16:38:27.369+11:00");
     public static final String VALID_TOKEN = "valid-token";
+    public static final String DB_E2E_1_TOKEN = "db_e2e_1";
```

Comment:
Can you re-use the VALID_TOKEN ? Mntner SSO uuid should be of test@ripe.net?

Reply:
Done